controllers/discovery/searchUser.js

```diff
@@ -82,6 +88,7 @@ const SearchUser = async (req, res) => {
         type: QueryTypes.SELECT,
         replacements: {
           likeQuery: '%' + q + '%',
+          allow_anon_dm,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `allow_anon_dm` variable is used in the SQL query without being sanitized first. This could potentially lead to SQL injection attacks. Always sanitize your variables before using them in SQL queries.
```diff
- allow_anon_dm,
+ allow_anon_dm: Boolean(allow_anon_dm),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->